docs/eslint.md

```diff
@@ -21,7 +21,7 @@ yarn add --dev prettier eslint-plugin-prettier
 
 ```json
 {
-  "plugins": ["prettier"],
+  "extends": ["plugin:prettier/recommended"],
   "rules": {
     "prettier/prettier": "error"
```

Comment:
remove this as well?

Reply:
Oops, didn't mean to change that block at all.